config/routes.rb

```diff
@@ -21,7 +21,11 @@
   resources :configurations, only: [:index, :update]
 
   resources :issues do
-    collection { post :import }
+    collection do
+      post :import
+      post :merging
```

Comment:
nothing happens yet when at the `:merging` correct? I think it's better to use `GET` at that step.